src/components/calendar/bl-calendar.constant.ts

```diff
@@ -1,14 +1,14 @@
 export const FIRST_MONTH_INDEX = 0;
 export const LAST_MONTH_INDEX = 11;
 
-export const CALENDAR_VIEWS = {
-  days: "days",
-  months: "months",
-  years: "years",
-} as const;
+export enum CALENDAR_VIEWS {
+  days = "days",
+  months = "months",
+  years = "years",
+}
```

Comment:
Can we use CONSTANT_CASE for enums?
```suggestion
export enum CALENDAR_VIEWS {
  DAYS = "days",
  MONTHS = "months",
  YEARS = "years",
}
```